thunder/core/langctxs.py

```diff
@@ -64,8 +64,17 @@ def reset_langctx(token: Any, /) -> None:
 
 # A helper for acquiring a method
 def resolve_method(id: Any, *args, **kwargs) -> Callable:
-    ctx: LanguageContext = get_langctx()
-    method: Callable = ctx.get_method(id, *args, **kwargs)
+    # I could combine the try/except statements but intentionally separated them for clarity
```

Comment:
Let's remove this comment. It makes sense, but most readers won't be interested in this detail